steps/generate_dataset_from_htme.py

```diff
@@ -39,6 +41,9 @@
 AUDIT_TABLE_RUN_ID_KEY = "Run_Id"
 AUDIT_TABLE_DATE_KEY = "Date"
 AUDIT_TABLE_STATUS_KEY = "Status"
+AUDIT_TABLE_CLUSTER_ID_KEY = "Cluster_Id"
+AUDIT_TABLE_CURRENT_STEP_KEY = "CurrentStep"
+AUDIT_TABLE_S3_PREFIX_KEY = "S3_Prefix"
```

Comment:
`DATA_PRODUCT_NAME` above, should it be ADG-FULL?

and also could we change these to match how the rest is populating the table/? 

IN_PROGRESS_STATUS = "In-Progress"
FAILED_STATUS = "FAILED"
COMPLETED_STATUS = "COMPLETED"

Reply:
No, it's not uppercase anywhere else, it's lower case, so just matched that. Fixed statuses.